firefox-ios/Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -790,11 +790,7 @@ class MainMenuActionHelper: PhotonActionSheetProtocol,
 
             let site = Site.createBasicSite(url: url.absoluteString, title: title)
 
-            self.profile.pinnedSites.addPinnedTopSite(site).uponQueue(.main) { result in
-                guard result.isSuccess else { return }
-                self.delegate?.showToast(message: .LegacyAppMenu.AddPinToShortcutsConfirmMessage, toastAction: .pinPage)
-            }
-
+            _ = self.profile.pinnedSites.addPinnedTopSite(site)
```

Comment:
I wonder if we could add `@discardableResult` on `addPinnedTopSite` and `removeFromPinnedTopSites` since we seem to not care about the result in a bunch of places in the codebase already. What do you think?

Reply:
I added `@discardableResult ` and fix the calls where the result was not needed. Thanks for the suggestion